resources/build/utils.inc.sh

```diff
@@ -297,9 +225,8 @@ check-markdown() {
 #   1: coverage_threshold   optional, minimum coverage for c8 to pass tests,
 #                           defaults to 90 (percent)
 #
-# Todo:
-#   Move to builder.typescript.inc.sh when this is established
-#
+# TODO: move to builder.typescript.inc.sh when this is established, rename to
+#       builder_typescript_do_tests
 builder_do_typescript_tests() {
```

Comment:
TODO: move `builder_do_typescript_tests` to builder.typescript.inc.sh, rename to `builder_typescript_do_tests`